### DISCUSSION TITLE
边上有Tools,类型是vertices，边上的双击编辑功能不灵敏该如何解决

### DISCUSSION THREAD

**Discussion by tanjianfeng123** [antvis/X6#3386] [Q&A]
边上有Tools,类型是vertices，边上的双击编辑功能不灵敏该如何解决

**Comment by Epic-Deno** [comment]
你是说edge边上双击编辑吗
1. 监听edge的双肩事件， 做个弹出框显示个表单，修改边的信息
![image](https://user-images.githubusercontent.com/74635349/226539162-b1f972d2-3c77-42b7-8a20-743e202e70ea.png)
2. 设置成功之后的回掉删掉边上原来的文本， 然后通过appendLabel插入新的文字
![image](https://user-images.githubusercontent.com/74635349/226539297-ec890e75-f24d-47d9-b16b-a7d85ac1aaab.png)
